tests/test_components.py

```diff
@@ -0,0 +1,145 @@
+import datetime
+import json
+import unittest
+
+from src import battery, loading_curve, scenario
+
+
+def get_test_json():
+    # get minimum working json example
+    return {
+        "scenario": {
+            "start_time": "2020-01-01T00:00:00+02:00",
+            "interval": 15,
+            "n_intervals": 35040
```

Comment:
What is your opinion, do we still need to simulate a full year?

Reply:
I don't see a reason why this is necessary. For what reason did you set it up like this?

Reply:
Seeing as nothing actually happens (no events), it does not really matter how long we simulate. For a minimal test, this could also be set to one day.